src/libraries/Microsoft.Extensions.Logging.Console/src/ConsoleLoggerProvider.cs

```diff
@@ -157,12 +157,14 @@ public ILogger CreateLogger(string name)
 #pragma warning disable CS0618
         private static void UpdateFormatterOptions(ConsoleFormatter formatter, ConsoleLoggerOptions deprecatedFromOptions)
         {
+            bool disableColors = deprecatedFromOptions.DisableColors || System.Console.IsOutputRedirected;
```

Comment:
```suggestion
            bool disableColors = deprecatedFromOptions.DisableColors || Console.IsOutputRedirected;
```

Reply:
`Console` here would get confused with `Microsoft.Extensions.Logging.Console` and because of that I need to specify `System.Console`. Otherwise gives error below:

```
> The type or namespace name 'IsOutputRedirected' does not exist in the namespace 'Microsoft.Extensions.Logging.Consol
e' (are you missing an assembly reference?)
```